src/logic/redux/rootReducer.js

```diff
@@ -0,0 +1,26 @@
+import reducers from 'logic/reducers';
+import { createActions } from 'reduxsauce';
+import { persistCombineReducers } from 'redux-persist';
+
+import { storeConfig } from './config';
+
+export const { Types } = createActions(
+  {
+    reset: null,
+  },
+  { prefix: 'state/' },
+);
+
+const appReducers = persistCombineReducers(storeConfig, {
+  ...reducers,
```

Comment:
I see you have added "@lagunovsky/redux-react-router", but there's no reducer for it